develop/jsapi/add_feature_layer/lab.md

```diff
@@ -0,0 +1,54 @@
+###Add a feature layer to a map
+
+In this lab you will add a feature layer to an ArcGIS API for JavaScript application. 
+
+1. Click [create_starter_map/index.html](../create_starter_map/index.html) and copy the contents to a new [jsbin.com](http://jsbin.com).
+
+2. In `JSBin` > `HTML`, update the `require` statement and function definition:
+
+  ```javascript
+  require([
+    "esri/Map",
+    "esri/views/MapView",
+    "esri/layers/FeatureLayer", // ADD FeatureLayer module
+    "dojo/domReady!"
+  ], function(Map, MapView, FeatureLayer) { // ADD FeatureLayer reference
+  ...
+  ```
+
+3. Now add a new `FeatureLayer` to the map:
+
+  ```javascript
+    ], function(Map, MapView, FeatureLayer) {
+
+      var map = new Map({
+        basemap: "dark-gray"
+      });
+
+      var view = new MapView({
+        container: "viewDiv",
+        map: map,
+        center: [-122.68, 45.52],
+        zoom: 10
+      });
+
+      // ADD Create a feature layer and add it to the map
```

Comment:
This is almost the same language as the instruction. I ran into this issue too. Should standardize.